pages/eligibility/index.tsx

```diff
@@ -230,4 +215,25 @@ const Eligibility: NextPage = () => {
   )
 }
 
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const query = context.query
+
+  const host = context.req.headers.host
+
+  const params = Object.keys(query)
+    .map((key) => key + '=' + query[key])
+    .join('&')
+
+  const path = `http://${host}/api/calculateEligibility?${params && params}`
```

Comment:
I tried changing this to just `${params}` (without the `&&`) and it seemed to work the same, so what's the reason behind this? Not sure what it's doing here.

Reply:
I was under the (incorrect) impression that if params was undefined it would interact badly with the string interpolation going on. To counteract this, I just did a truthiness check that would make sure it worked then append the string. This was some major overengineering on my part and was how I had it with client side rendering.

Fixed in my next commit!